calico_kubernetes/calico_kubernetes.py

```diff
@@ -257,17 +258,42 @@ def _generate_rules(self):
 
         :return list() rules: the rules to be added to the Profile.
         """
+
+        namespace, ns_tag = self._get_namespace_and_tag(pod)
+
         inbound_rules = [
             {
-                'action': 'allow',
-            },
+                "action": "allow",
+                "src_tag": ns_tag
+            }
         ]
 
         outbound_rules = [
             {
-                'action': 'allow',
-            },
+                "action": "allow"
+            }
         ]
+
+        print('Getting Policy Rules from Annotation of pod %s' % pod)
+
+        annotations = self._get_metadata(pod, 'annotations')
+
+        if 'allowFrom' in annotations.keys():
+            inbound_rules = []
```

Comment:
Stick a comment here about why we're blatting over the other inbound_rules, so we don't forget.